jira/cli/templates.go

```diff
@@ -52,12 +52,18 @@ comments:
 `
 const default_edit_template = `# issue: {{ .key }}
 update:
+{{if .overrides.comment}}
```

Comment:
The goal here is to auto populate the comment if someone used -m "my comment" but still allow someone to manually type in a comment while running "jira edit".  So in the case that someone wants to add a comment but does not use "jira edit -m ..." that section of the template needs to exist, so I think it should still look something like:
```
comment:
  - add:
    body: |~
       {{ or .overrides.comment "" | indent 10 }}
```